shared/src/store/action-reducers/exercise.ts

```diff
@@ -125,6 +131,12 @@ export namespace ExerciseActionReducers {
                     currentPatient.visibleStatusChanged
                 ) {
                     updateTreatments(draftState, currentPatient);
+                    logPatient(
+                        draftState,
+                        [],
+                        `Die Sichtungskategorie von ${currentPatient.personalInformation.name} hat sich geändert.`,
```

Comment:
You're inconsistent whether you include the patient name in the message or not. For example, you don't include the name in the treatment reassignment log messages.

By the way, using the name does not make that much sense, as there is currently no way to find out the name of a patient who is part of the simulation (except of moving it to the map, opening the popup, reading the name and putting it back to the simulated region). So maybe we should implement #1022 (was just an idea in our internal list for a couple of weeks but it seems to be worth implementing it now).

Reply:
What do you think about the mentioned issue?

Reply:
We've decided to work on the issue so this PR can be merged as it is for now and the tags will be updated when the new IDs are ready